.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.15.x]
+        go-version: [1.16.0-beta1, 1.15.7, 1.14.14]
```

Comment:
We decided to use 1.15.7 as oldest, see https://github.com/mattfarina/hypper/pull/11#discussion_r561987772.

Reply:
That seems to be 1.15 vs 1.13 but supported versions for go are release and release minus 1, so Im guessing we want to have that as well? Like we want to make sure we cover both stable releases ,plus Im sure down the line the SLES supported go version its on the requirements and that sometimes lag a bit compared to the latest go version :)

Reply:
just to have it written down, the reason for using 1.15.X only was a security bug vs 1.13.X

That bug is fixed in 1.14.14 so it should be safe to use.

Reply:
Sounds good, resolving the conversation :).